### ISSUE TITLE
same output with different inputs

### ISSUE THREAD

**Issue by twowwj** [eladrich/pixel2style2pixel#273]
Hi yuval-alaluf,

As the title described, I use the psp_encoder to generate the latent code of Low-resolution image with the size of 64, which processed by the transforms.Resize([256,256], interpolation=Image.BICUBIC).
![4_lr  2](https://user-images.githubusercontent.com/33343889/170376440-b425be8c-6f9e-4f4e-88d9-f8c1a0ce897b.png)
and use the styleganv2 generator to generate the image. But I have found that if I use the lr image(64*64) to input the psp_encoder, the outputs are always the same person.
![4_psp  8](https://user-images.githubusercontent.com/33343889/170376245-d49bce5b-1fd5-44ad-b954-2afe2b8a5506.png)
If I upsample the lr image to the size of 256, it seems normal but blurred.
![5_psp  2](https://user-images.githubusercontent.com/33343889/170382393-fff4af7c-a0be-46e3-9349-bf0d69b07042.png)
Do you have exact suggestions for me?
here is my code about psp_encoder, are I wrong to use it?
![image](https://user-images.githubusercontent.com/33343889/170377109-e07078d1-3ae0-45e7-8b9b-f0fd7a4a7a63.png)

thx

**Comment by twowwj** [comment]
![image](https://user-images.githubusercontent.com/33343889/170380837-04d34318-0f7e-4bd3-9396-f433d0d42bb0.png)
The right picture is the output of the psp_encoder, It seems not very good, am I wrong? 
sorry for the bother.

thx

**Comment by yuval-alaluf** [comment]
Are you using the standard pSp encoder? If so, the model was trained on the FFHQ dataset which contains high-quality images of size 256x256. Therefore, if you pass an image that was of size 64x64 and then upsample it to 256x256, you will get a low quality image like the one of the man on the left in your second comment. 
Have you tried using the pSp super resolution model instead?

**Comment by twowwj** [comment]
I have made some tests, if I input the image of size 64*64, It will always give me the same person, am I right?
and I have used the psp super resolution model to generate high resolution image, it seems everything is okay. 
The quality of image is better than the standard psp encoder.
![0_psp](https://user-images.githubusercontent.com/33343889/170816018-e4bef357-5078-4f87-96ec-1b03076528f6.png)
![0_psp](https://user-images.githubusercontent.com/33343889/170816026-a2171dd2-e332-43d5-ac71-b9cf26e85b96.png)

In the middle, it is the result of standard psp.
At the bottom, it is the result of super resolution psp.
I think I should use the super resolution model of psp.
